source/server/configuration_impl.cc

```diff
@@ -54,6 +54,21 @@ void FilterChainUtility::buildUdpFilterChain(
   }
 }
 
+StatsConfigImpl::StatsConfigImpl(const envoy::config::bootstrap::v3::Bootstrap& bootstrap) {
+  if (bootstrap.has_stats_flush_interval() &&
+      bootstrap.stats_flush_case() !=
+          envoy::config::bootstrap::v3::Bootstrap::STATS_FLUSH_NOT_SET) {
+    throw EnvoyException("Only one of stats_flush_interval or stats_flush_on_admin should be set!");
```

Comment:
Please add a test for this case.

Reply:
There's a test case for this in configuration_impl_test, `IntervalAndAdminFlush`.